app/src/main/java/org/mifos/mobilebanking/presenters/AccountsPresenter.java

```diff
@@ -148,9 +149,9 @@ public void onNext(ClientAccounts clientAccounts) {
     public List<SavingAccount> searchInSavingsList(List<SavingAccount> accounts,
                                             final String input) {
         return Observable.from(accounts)
```

Comment:
@therajanmaurya `Observable.from` is having problems now. How can I solve this? I tried `Observable.fromIterable` but it didn't worked.

Reply:
@Rish02Sharma 
`filter(new Predicate<SavingAccount>() {`
                    `@Override`
                    `public boolean test(SavingAccount savingAccount) throws Exception {`
                        `return false;`
                    `}`
               ` })`
`fromIterable` should work. `filter` in `RxJava2` accepts a predicate not a function
I think that is the issue here
Can you please check

Reply:
@mohak1712 Yeah I tried this already. But no luck here! :/

Reply:
@Rish02Sharma please try this 
```
return Observable.fromIterable(accounts)
                .filter(new Predicate<SavingAccount>() {
                    @Override
                    public boolean test(SavingAccount savingAccount) throws Exception {
                        return false;
                    }
                }).toList().blockingGet();
```